src/VirtoCommerce.Platform.Web/wwwroot/js/i18n/resources/countries.js

```diff
@@ -1,273 +1,281 @@
-﻿// Full list of countries defined by ISO 3166-1 alpha-3
+// Full list of countries defined by ISO 3166-1 alpha-3
 // based on https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3
 angular.module('platformWebApp')
-.factory('platformWebApp.common.countries', function () {
-    return {
-        get: function (id) {
-            return _.findWhere(this.query(), { id: this.normalize(id) });
-        },
-        contains: function (id) {
-            return _.map(this.query(), function (entry) { return entry.id }).includes(this.normalize(id));
-        },
-        normalize: function(id) {
-            var result = undefined;
-            if (!!id) {
-                result = id.toUpperCase();
+    .factory('platformWebApp.common.countries', ['$translate', function ($translate) {
+        return {
+            get: function (id) {
+                return _.findWhere(this.query(), { id: this.normalize(id) });
+            },
+            contains: function (id) {
+                return _.map(this.query(), function (entry) { return entry.id }).includes(this.normalize(id));
+            },
+            normalize: function (id) {
+                var result = undefined;
+                if (!!id) {
+                    result = id.toUpperCase();
+                }
+                return result;
+            },
+            query: function () {
+                var results = [
```

Comment:
Why this list is not deleted from the platform?

Reply:
It's still the primary source of countries. Any requirement to refactor?

Reply:
![image](https://user-images.githubusercontent.com/47894313/120296575-8d9c2380-c2c8-11eb-9beb-aa37fbe1b7f0.png)
With this requirement, we don't this list in the JS file.

Reply:
I agree with @Vectorfield4, why do we still need the hard-coded country list in this file? Why don't you move the list of countries for the English language to the default resource file, and read it as an array here instead?

Reply:
I'm pretty sure $translate has the ability to read resource arrays for a specific path. If not, we can always read localization resources as JSON.

Reply:
API added